peps/pep-9999.rst

```diff
@@ -0,0 +1,193 @@
+
+The working goup will maintain
+documentation, test suites and tooling related to Python's C API.
+It is the deciding body for changes to the C API,
+from the addition or removal of individual API functions, types, etc.,
+to the the acceptance of new designs of a more or less radical nature.
+
+The working goup's mandate is to represent the interests of all Python users,
+but especially all maintainers of code that uses Python's C API,
+whether in the context of CPython, using an alternate Python implementation,
+or using a binding framework for other languages.
+
+The working group serves at the pleasure of the Python Steering Council.
+The working group's members are the listed authors of this PEP.
+This document serves as the working group's charter.
+
+Epigraph
+========
+
+| KEEPER
+| Stop!
+  Who would cross the Bridge of Death must answer me these questions three,
+  ere the other side he see.
+
+#. Who invented Python?
+#. In what year did Python 2.7 reach its End-of-Life?
+#. What is the future of the Python C API?
+
+| LANCELOT
+| Auuuuuuuugh!
+
+Motivation
+==========
+
+Despite many discussions and in-person meetings
+at core developer sprints and Language Summits,
+and a thorough inventory of the problems and stakeholders of the C API,
+no consensus has been reached about many contentious issues,
+including, but not limted to:
+
+- Conventions for designing new API functions;
+- how to deal with compatibility;
+- what's the best strategy for handling errors;
+- the future of the Stable ABI and the Limited API;
+- whether to switch to a handle-based API convention (and how).
+
+The general feeling is that there are too many stakeholders,
+proposals, requirements, constraints, and conventions,
+to make progress without having a small trusted group of deciders.
+
+At the 2023 Language Summit in Salt Lake City it was decided to start work on
+an `inventory of problems <https://github.com/capi-working-group/problems>`__.
+At the 2023 core developer sprint in Brno this work is more or less finished
+and after a discussion it appeared that the next step is to establish
+a working group to ensure that we're not stymied forever.
+
+The Steering Council has
+`indicated <https://github.com/python/steering-council/issues/201#issuecomment-1648848155>`__
+its desire to delegate decisions about the C API
+to such a working group, anticipating its formal establishment.
+
+Specification
+=============
+
+We propose the creation of a new group, the C API Working Group.
+This group will be responsible for overseeing and coordinating the development and
+maintenance of the Python C API.
+It will do this by establishing the principles underpinning the work
+and publishing guidelines that the core developers can refer to.
+
+The "operations and process" section below describes
+how the working group operates and how it is governed.
+
+Mandate
+-------
+
+The working group's mandate is to ensure that the Python C API is:
```

Comment:
```suggestion
The working group's mandate is to ensure that the Python C API is suitable for
all users and contributors to the API, without unduly preferencing one group
over another. The working group will identify exemplar stakeholders, their
needs and preferences, and will determine a plan for meeting these needs
equitably and sustainably.
```

Maybe?

Reply:
Oddly, this also feels a bit specific. What does the WG do once we're done making a plan? More below.

Reply:
Executes the plan :-)

I think my main aim is to define a scope here, but not the plan itself, other than to require inclusiveness and fairness (without strictly defining who/what/why/where/etc.). We've done a lot of analysis so far, but if we do a proper summary and report on that, we shouldn't need to update this PEP if we decide on a slightly different path.

Reply:
Okay, I'll try to wordsmith your suggestion a bit to imply that.